lib/scoped.js

```diff
@@ -0,0 +1,156 @@
+module.exports = scoped
+
+var merge = require('lodash.merge')
+var EventEmitter = require('events').EventEmitter
+var toId = require('pouchdb-hoodie-api/utils/to-id')
+
+function scoped (api, type) {
+  var emitter = new EventEmitter()
+
+  var scopedApi = merge(
+    {},
+    api,
```

Comment:
I wasn't sure how much of the 'Store' API was being scoped so thanks for catching that. I'll refactor that to create just the methods we need.